components/feature/readerview/src/main/java/mozilla/components/feature/readerview/view/ReaderViewControlsBar.kt

```diff
@@ -111,6 +109,16 @@ class ReaderViewControlsBar @JvmOverloads constructor(
         visibility = View.GONE
     }
 
+    override fun tryInflate(): Boolean {
```

Comment:
nit: can we comment this is done for Fenix performance reasons: I.e. it's not created until it's needed? Not sure if this is the best place for it.

I wonder if "not inflated until necessary" would be useful in the class kdoc (maybe not, as an implementation detail).

Reply:
I had docs on the API, but I copied them over here as well.